velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -226,11 +252,10 @@ void Writer::write(const VectorPtr& data) {
       auto recordBatch, ::arrow::ImportRecordBatch(&array, &schema));
   if (!arrowContext_->schema) {
```

Comment:
@Yuhta, can the encodings across recordBatches vary? In that case, should we get the schema for every batch?